wvlet-cli/src/test/scala/wvlet/lang/cli/QueryConverterCommandTest.scala

```diff
@@ -0,0 +1,145 @@
+package wvlet.lang.cli
+
+import wvlet.airspec.AirSpec
+import wvlet.log.io.IOUtil
+
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+class QueryConverterCommandTest extends AirSpec:
+  // TPC-H query paths
+  private val tpchWvletQueryPath = "spec/tpch"
+  private val tpchSQLQueryPath = "spec/sql/tpc-h"
+
+  test("TPC-H query files exist") {
+    import java.io.File
+    for (i <- 1 to 22) {
```

Comment:
SourceIO.listSourceFiles("spec/tpch") or SourceIO.listSourceFiles("spec/sql/tpc-h") can be used for listing all .wv or .sql files.